RecoLocalCalo/HGCalRecProducers/plugins/HGCalUncalibRecHitProducer.cc

```diff
@@ -12,7 +12,7 @@
 HGCalUncalibRecHitProducer::HGCalUncalibRecHitProducer(const edm::ParameterSet& ps) :
   eeDigiCollection_( consumes<HGCEEDigiCollection>( ps.getParameter<edm::InputTag>("HGCEEdigiCollection") ) ),
   hefDigiCollection_( consumes<HGCHEDigiCollection>( ps.getParameter<edm::InputTag>("HGCHEFdigiCollection") ) ),
-  hebDigiCollection_( consumes<HGCHEDigiCollection>( ps.getParameter<edm::InputTag>("HGCHEBdigiCollection") ) ),
+  hebDigiCollection_( consumes<HGCBHDigiCollection>( ps.getParameter<edm::InputTag>("HGCHEBdigiCollection") ) ),
```

Comment:
EE = EE
HEF = FH
HEB = BH

The left are old naming schemes that have unfortunately suck. 
I would like to sanitize/modernize them, but in another PR.